Gemfile

```diff
@@ -2,7 +2,7 @@ source 'https://rubygems.org'
 
 gem 'sinatra'
 
-gem 'rmagick'
+gem 'rmagick', '~> 2.15.4'
```

Comment:
Is there something special about this version? If so, mind adding a comment? If not, I prefer to keep the version restriction out of the Gemfile.